File: 04GuessTheNumber/GuessTheNumber.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
  int inputNum;
  const int MIN = 1;
  const int MAX = 1000;

  srand(time(NULL));
  int randomNum = (rand() % MAX) + MIN;

  while (1) {
    printf("Give me a number between 1 to 1000: ");
    scanf("%d", &inputNum);

    if (inputNum >= MIN && inputNum <= MAX) {
      if (inputNum == randomNum) {
        printf("Congratulations, You're now officially toddler.c\n");
        break;
      } else if (inputNum > randomNum) {
        printf("Too High!\n");
      } else {
        printf("Too Low!\n");
      }
    } else {
      printf("Invalid input dawg! Check Again!!\n");
      break;
    }
  }

  return 0;
}

File: 04GuessTheNumber/GuessTheNumber.py
import random


def mainFunc():
    randomInteger = random.randint(1, 1000)

    while True:
        try:
            inputNum = int(input("Give me a Number between 1 and 1000 : "))
        except ValueError:
            print("Invalid input dawg! try again")
            continue

        if inputNum == randomInteger:
            print("Congratulations, you're an officially toddler.py")
            break
        elif inputNum > randomInteger:
            print("Too High")
        else:
            print("Too Low")


mainFunc()
